### ISSUE TITLE
context.query not resolved correctly

### ISSUE THREAD

**Issue by germsb** [vueneue/vueneue#6]
`context.query` is resolved server side but not after navigate client side

```
async asyncData({query, params}) {
      console.log('query: ', query, ' - params: ', params)
}
```
server : 
```query:  { test: 'ncdchu' }  - params:  { lang: 'fr' }```
client :
```query:  {}  - params:  { lang: 'fr' }```

**Comment by chymz** [comment]
Thanks for reporting this ! It will be fixed in next release.

**Comment by chymz** [comment]
Fixed in `0.3.7`
